front/app/containers/Admin/formBuilder/components/FormBuilderToolbox/ToolboxItem.tsx

```diff
@@ -13,31 +13,42 @@ interface Props {
   onClick: () => void;
 }
 
-const StyledBox = styled(Box)`
+const AddIcon = styled(Icon).attrs({ name: 'plus', ml: '20px' })`
+  width: 16px;
+  height: 16px;
+  fill: ${colors.adminSecondaryTextColor};
+`;
+
+const StyledButton = styled(Button)`
+  ${AddIcon} {
+    visibility: hidden;
+  }
   &:hover {
     background-color: ${colors.emailBg};
```

Comment:
Missed this earlier, but having "emailBg" colour here seems strange to me. If it's that colour that Ben would like, maybe we can rename it or add another named colour in the future which makes more sense in this context?

Reply:
I see it's used in the content builder as well - maybe we can bring this up with Ben when he's back :)

Reply:
That makes sense. I have created this task ticket to track it https://citizenlab.atlassian.net/browse/CL-1364